modules/kafka-cluster/outputs.tf

```
@@ -10,7 +10,7 @@ output "kafka_cluster_id" {
 
 output "kafka_cluster_type" {
   description = "Type of Cluster"
-  value       = (var.cluster_for_production) ? "basic" : "standard"
+  value       = var.cluster_type
```

I think is better if we read this out from the resource `confluent_kafka_cluster` instead of from the variable. The reason is that, if outside this component module someone is reading this output `kafka_cluster_type`, Terraform will make the output immediately available before the resource `confluent_kafka_cluster` is built. This leads to some error in the form of the consumer trying to use `confluent_kafka_cluster` but `confluent_kafka_cluster` is not built yet.

```suggestion
  value       =  length(confluent_kafka_cluster.cluster.basic) > 0 ? "basic" : (length(confluent_kafka_cluster.cluster.standard) > 0 ? "standard" : "dedicated" )
```

---

thanks I have updated the output value